Makefile

```diff
@@ -5,27 +5,61 @@ MANPAGE = genkernel.8
 # Add off-Git/generated files here that need to be shipped with releases
 EXTRA_DIST = $(MANPAGE) ChangeLog $(KCONF)
 
-default: kconfig man
```

Comment:
keep; likely change to:
`default: all`
For non-GNU make engines if they are used

Reply:
addressed in commit [e87c328](https://github.com/gentoo/genkernel/commit/e87c328eac10865f4976cf2f96d3a5f86bbfe88c)